### DISCUSSION TITLE
Started getting Jellyseer errors today

### DISCUSSION THREAD

**Discussion by j2ther** [seerr-team/seerr#353] [Q&A] [answered]
Went to go use Jellyseer today and was getting 500 internal server error on the web.  Looking at powershell I get this list of errors

https://pastebin.com/R0CuYpdd

Using Jellyseer in windows 10.  Any help is appreciated!

**Comment by fallenbagel** [comment, marked as answer]
How are you running it? It seems its unable to access tmdb

**Reply by j2ther** [reply]
Your post made me realize that I had it behind a vpn without acess to the wan.  Dumb moment, sorry, it's all working just fine now.

**Comment by Jenil-Desai** [comment]
i am getting same issue what to do?

**Reply by fallenbagel** [reply]
Could you share your logs. Also best to hop onto discord so its easier to follow as we do support there

**Reply by Jenil-Desai** [reply]
LOGS :https://pastebin.com/Ug1ghbyT

I am hosting it in docker container with image version as follow : v1.5.0

**Reply by fallenbagel** [reply]
Your container is not connecting to the Internet. Could you hop onto discord so we can follow through?

**Reply by Jenil-Desai** [reply]
NO PROBLEM DISCORD ID?

**Reply by fallenbagel** [reply]
> NO PROBLEM DISCORD ID?
> 

It's In the readme.md the invite link
